From hjk@linutronix.de  Fri Dec 21 12:13:01 2007
From: Hans-Jürgen Koch <hjk@linutronix.de>
Date: Wed, 5 Dec 2007 15:08:07 +0100
Subject: UIO: Add a MAINTAINERS entry for Userspace I/O
To: Greg KH <greg@kroah.com>
Message-ID: <20071205150807.14cc818d@dilbert.local>

From: Hans-Jürgen Koch <hjk@linutronix.de>

This patch adds an entry for the Userspace I/O framework to MAINTAINERS.

Signed-off-by: Hans J. Koch <hjk@linutronix.de>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


---
 MAINTAINERS |    8 ++++++++
 1 file changed, 8 insertions(+)

--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -4054,6 +4054,14 @@ L:	user-mode-linux-user@lists.sourceforg
 W:	http://user-mode-linux.sourceforge.net
 S:	Maintained
 
+USERSPACE I/O (UIO)
+P:	Hans J. Koch
+M:	hjk@linutronix.de
+P:	Greg Kroah-Hartman
+M:	gregkh@suse.de
+L:	linux-kernel@vger.kernel.org
+S:	Maintained
+
 FAT/VFAT/MSDOS FILESYSTEM:
 P:	OGAWA Hirofumi
 M:	hirofumi@mail.parknet.co.jp
